Release 1.0.0

abelspry opened the pull request:
- Developer(s): @vhrychenko @abelspry 

#### Please confirm

- [ ] No new OS components - or they have been approved by the legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   ShopUiCompatibility   | major (new)           |                       |

#### Release Notes

The release version of shop-ui-compatibility module

-----------------------------------------

#### Module ShopUiCompatibility

_**Mini-Major:** Added functionality with incompatible API changes_

Def of done (by the responsible developer):
- [ ] API methods are defined and specified [API-doc](https://academy.spryker.com/understanding_spryker/definition_api.html).
- [ ] New and changed facade methods covered by functional tests. / Has nothing to test.
- [ ] New and changed complex business logic is covered by unit or integration tests. / Has nothing to test.
- [ ] Cleanups are resolved. / No relevant cleanups. [cleanups](http://release-app.spryker-shop-b2c.com/module-cleanups).
- [ ] Migration guide provided.

##### Change log

Initial Release

The release version of shop-ui-compatibility module

Commit:
add missing assets and widgetGlobalExists function

Commit:
adapt module to spryker-eco configuration

Commit:
update composer.json

Commit:
fix ShopUiCompatibilityDefineTwigNode parse

Commit:
rename folder builder into build

Commit:
remove core-detector feature

pugach-ivan committed:
Improve module

pugach-ivan committed:
Merge branch 'feature/eco-and-css-namespace-full-integration' of https://github.com/spryker-eco/shop-ui-compatibility into feature/eco-and-css-namespace-full-integration

tolerants commented on assets/Yves/build/configs/development.js:
```diff
@@ -0,0 +1,155 @@
+        new CopyWebpackPlugin([
+            {
+                from: `${appSettings.paths.assets}/images/*`,
+                to: './images/[name].[ext]',
```
Fix from https://spryker.atlassian.net/browse/CC-556 should be applied.

tolerants replied:
Fixed.

vhrychenko committed:
Merge branch 'master' into feature/eco-and-css-namespace-full-integration

# Conflicts:
#	composer.json

vhrychenko committed:
some fixes before release

vhrychenko committed:
sniffer fixes

vhrychenko committed:
end of line

vhrychenko committed:
Fix travis

vhrychenko committed:
Fix phpstan and update factory

vhrychenko committed:
Update license

Commit:
CC-485: Checked and prepared for release.
- Fixed small issues with assets compilation
- Added module description.
- Removed package-lock.json.
- Increased module version to 1.0.0.

Commit:
CC-485: Fixed description in README.md.
